wgpu/src/lib.rs

```diff
@@ -1687,9 +1687,11 @@ impl Adapter {
 impl Device {
     /// Check for resource cleanups and mapping callbacks.
     ///
+    /// Return `queue_empty` indicating whether there are more queue submissions still in flight.
+    ///
     /// no-op on the web, device is automatically polled.
-    pub fn poll(&self, maintain: Maintain) {
-        Context::device_poll(&*self.context, &self.id, maintain);
+    pub fn poll(&self, maintain: Maintain) -> bool {
+        Context::device_poll(&*self.context, &self.id, maintain)
```

Comment:
Should we make a similar change to `Instance::poll_all`?

Reply:
Yeah, we should do that.